bin/reservation.js

```diff
@@ -1,4 +1,4 @@
-let table_name = 'Reservation'
+let table_name = 'reservation'
```

Comment:
I think you was trapped with this names :)
This is why i follow this rule - Before i write like `reservationModel` - just to be sure that model name is called.
But then i put model names in datasource.json and config like `Reservation`. So each time when i using a model methods in code - i know that `Reservation.find(....)` related to db model. maybe this will helps

Reply:
cool thanks for the tip :).